tests/adapter/freshheads/CssAdapter.test.js

```diff
@@ -43,9 +43,6 @@ describe('FreshheadsCssAdapter', () => {
                     loader: 'css-loader',
                     options: {
                         sourceMap: true,
-                        modules: {
-                            auto: true,
```

Comment:
is nu default met css-loader 4.x, daarnaast worden css-modules nu beter getreeshaked omdat ze als es modules geimporteerd kunnen worden.